[2024-02-06T22:19:30.608+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Competition_dag.cleanMatch_task scheduled__2024-01-03T00:00:00+00:00 [queued]>
[2024-02-06T22:19:30.645+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Competition_dag.cleanMatch_task scheduled__2024-01-03T00:00:00+00:00 [queued]>
[2024-02-06T22:19:30.656+0100] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-02-06T22:19:30.784+0100] {taskinstance.py:2191} INFO - Executing <Task(SparkSubmitOperator): cleanMatch_task> on 2024-01-03 00:00:00+00:00
[2024-02-06T22:19:30.834+0100] {standard_task_runner.py:60} INFO - Started process 44204 to run task
[2024-02-06T22:19:30.933+0100] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'Competition_dag', 'cleanMatch_task', 'scheduled__2024-01-03T00:00:00+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/datalake_dag.py', '--cfg-path', '/tmp/tmpgs0d63zr']
[2024-02-06T22:19:30.949+0100] {standard_task_runner.py:88} INFO - Job 33: Subtask cleanMatch_task
[2024-02-06T22:19:31.232+0100] {task_command.py:423} INFO - Running <TaskInstance: Competition_dag.cleanMatch_task scheduled__2024-01-03T00:00:00+00:00 [running]> on host ubuntu.ubuntu.virtualbox.org
[2024-02-06T22:19:31.716+0100] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='NOUMEN_Frech_Warren' AIRFLOW_CTX_DAG_ID='Competition_dag' AIRFLOW_CTX_TASK_ID='cleanMatch_task' AIRFLOW_CTX_EXECUTION_DATE='2024-01-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-03T00:00:00+00:00'
[2024-02-06T22:19:31.746+0100] {base.py:83} INFO - Using connection ID 'spark_default' for task execution.
[2024-02-06T22:19:31.748+0100] {spark_submit.py:401} INFO - Spark-Submit cmd: spark-submit --master yarn --conf spark.airflow.execution_date=2024-01-03 --name arrow-spark --class FormattedData --queue root.default /home/ubuntu/airflow/scala/target/scala-2.13/scala_2.13-0.1.0.jar
[2024-02-06T22:19:41.756+0100] {spark_submit.py:571} INFO - 24/02/06 22:19:41 WARN Utils: Your hostname, ubuntu resolves to a loopback address: 127.0.1.1; using 10.0.2.15 instead (on interface enp0s3)
[2024-02-06T22:19:41.767+0100] {spark_submit.py:571} INFO - 24/02/06 22:19:41 WARN Utils: Set SPARK_LOCAL_IP if you need to bind to another address
[2024-02-06T22:19:41.985+0100] {spark_submit.py:571} INFO - Exception in thread "main" org.apache.spark.SparkException: When running with master 'yarn' either HADOOP_CONF_DIR or YARN_CONF_DIR must be set in the environment.
[2024-02-06T22:19:41.986+0100] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmitArguments.error(SparkSubmitArguments.scala:634)
[2024-02-06T22:19:41.987+0100] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmitArguments.validateSubmitArguments(SparkSubmitArguments.scala:274)
[2024-02-06T22:19:41.988+0100] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmitArguments.validateArguments(SparkSubmitArguments.scala:234)
[2024-02-06T22:19:41.988+0100] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmitArguments.<init>(SparkSubmitArguments.scala:119)
[2024-02-06T22:19:41.988+0100] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit$$anon$2$$anon$3.<init>(SparkSubmit.scala:1055)
[2024-02-06T22:19:41.989+0100] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit$$anon$2.parseArguments(SparkSubmit.scala:1055)
[2024-02-06T22:19:41.989+0100] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit.doSubmit(SparkSubmit.scala:85)
[2024-02-06T22:19:42.007+0100] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit$$anon$2.doSubmit(SparkSubmit.scala:1072)
[2024-02-06T22:19:42.007+0100] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit$.main(SparkSubmit.scala:1081)
[2024-02-06T22:19:42.008+0100] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit.main(SparkSubmit.scala)
[2024-02-06T22:19:42.165+0100] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self._application)
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 502, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master yarn --conf spark.airflow.execution_date=2024-01-03 --name arrow-spark --class FormattedData --queue root.default /home/ubuntu/airflow/scala/target/scala-2.13/scala_2.13-0.1.0.jar. Error code is: 1.
[2024-02-06T22:19:42.169+0100] {taskinstance.py:1138} INFO - Marking task as UP_FOR_RETRY. dag_id=Competition_dag, task_id=cleanMatch_task, execution_date=20240103T000000, start_date=20240206T211930, end_date=20240206T211942
[2024-02-06T22:19:42.204+0100] {standard_task_runner.py:107} ERROR - Failed to execute job 33 for task cleanMatch_task (Cannot execute: spark-submit --master yarn --conf spark.airflow.execution_date=2024-01-03 --name arrow-spark --class FormattedData --queue root.default /home/ubuntu/airflow/scala/target/scala-2.13/scala_2.13-0.1.0.jar. Error code is: 1.; 44204)
[2024-02-06T22:19:42.314+0100] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-06T22:19:42.387+0100] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-02-06T22:34:16.943+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Competition_dag.cleanMatch_task scheduled__2024-01-03T00:00:00+00:00 [queued]>
[2024-02-06T22:34:17.001+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Competition_dag.cleanMatch_task scheduled__2024-01-03T00:00:00+00:00 [queued]>
[2024-02-06T22:34:17.002+0100] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-02-06T22:34:17.129+0100] {taskinstance.py:2191} INFO - Executing <Task(SparkSubmitOperator): cleanMatch_task> on 2024-01-03 00:00:00+00:00
[2024-02-06T22:34:17.153+0100] {standard_task_runner.py:60} INFO - Started process 54944 to run task
[2024-02-06T22:34:17.164+0100] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'Competition_dag', 'cleanMatch_task', 'scheduled__2024-01-03T00:00:00+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/datalake_dag.py', '--cfg-path', '/tmp/tmprmi124h2']
[2024-02-06T22:34:17.171+0100] {standard_task_runner.py:88} INFO - Job 36: Subtask cleanMatch_task
[2024-02-06T22:34:17.364+0100] {task_command.py:423} INFO - Running <TaskInstance: Competition_dag.cleanMatch_task scheduled__2024-01-03T00:00:00+00:00 [running]> on host ubuntu.ubuntu.virtualbox.org
[2024-02-06T22:34:17.715+0100] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='NOUMEN_Frech_Warren' AIRFLOW_CTX_DAG_ID='Competition_dag' AIRFLOW_CTX_TASK_ID='cleanMatch_task' AIRFLOW_CTX_EXECUTION_DATE='2024-01-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-03T00:00:00+00:00'
[2024-02-06T22:34:17.762+0100] {base.py:83} INFO - Using connection ID 'spark_default' for task execution.
[2024-02-06T22:34:17.796+0100] {spark_submit.py:401} INFO - Spark-Submit cmd: spark-submit --master local[*] --conf spark.airflow.execution_date=2024-01-03 --name arrow-spark --class FormattedData --queue root.default --deploy-mode client /home/ubuntu/airflow/scala/target/scala-2.13/scala_2.13-0.1.0.jar
[2024-02-06T22:34:28.530+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:28 WARN Utils: Your hostname, ubuntu resolves to a loopback address: 127.0.1.1; using 10.0.2.15 instead (on interface enp0s3)
[2024-02-06T22:34:28.548+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:28 WARN Utils: Set SPARK_LOCAL_IP if you need to bind to another address
[2024-02-06T22:34:31.222+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:31 INFO SparkContext: Running Spark version 3.3.4
[2024-02-06T22:34:31.492+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:31 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-02-06T22:34:32.087+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:32 INFO ResourceUtils: ==============================================================
[2024-02-06T22:34:32.091+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:32 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-02-06T22:34:32.095+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:32 INFO ResourceUtils: ==============================================================
[2024-02-06T22:34:32.112+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:32 INFO SparkContext: Submitted application: JobFormatted
[2024-02-06T22:34:32.222+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:32 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-02-06T22:34:32.267+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:32 INFO ResourceProfile: Limiting resource is cpu
[2024-02-06T22:34:32.270+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:32 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-02-06T22:34:32.611+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:32 INFO SecurityManager: Changing view acls to: ubuntu
[2024-02-06T22:34:32.619+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:32 INFO SecurityManager: Changing modify acls to: ubuntu
[2024-02-06T22:34:32.621+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:32 INFO SecurityManager: Changing view acls groups to:
[2024-02-06T22:34:32.625+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:32 INFO SecurityManager: Changing modify acls groups to:
[2024-02-06T22:34:32.643+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:32 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(ubuntu); groups with view permissions: Set(); users  with modify permissions: Set(ubuntu); groups with modify permissions: Set()
[2024-02-06T22:34:34.654+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:34 INFO Utils: Successfully started service 'sparkDriver' on port 41161.
[2024-02-06T22:34:35.138+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:35 INFO SparkEnv: Registering MapOutputTracker
[2024-02-06T22:34:35.325+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:35 INFO SparkEnv: Registering BlockManagerMaster
[2024-02-06T22:34:35.447+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:35 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-02-06T22:34:35.450+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:35 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-02-06T22:34:35.471+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:35 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-02-06T22:34:35.604+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:35 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-e905402d-2c89-4348-bc50-d0ce5975ebef
[2024-02-06T22:34:35.719+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:35 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-02-06T22:34:35.768+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:35 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-02-06T22:34:36.857+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:36 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-02-06T22:34:37.083+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:37 INFO SparkContext: Added JAR file:/home/ubuntu/airflow/scala/target/scala-2.13/scala_2.13-0.1.0.jar at spark://10.0.2.15:41161/jars/scala_2.13-0.1.0.jar with timestamp 1707255271189
[2024-02-06T22:34:37.755+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:37 INFO Executor: Starting executor ID driver on host 10.0.2.15
[2024-02-06T22:34:37.838+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:37 INFO Executor: Starting executor with user classpath (userClassPathFirst = false): ''
[2024-02-06T22:34:38.017+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:38 INFO Executor: Fetching spark://10.0.2.15:41161/jars/scala_2.13-0.1.0.jar with timestamp 1707255271189
[2024-02-06T22:34:38.568+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:38 INFO TransportClientFactory: Successfully created connection to /10.0.2.15:41161 after 304 ms (0 ms spent in bootstraps)
[2024-02-06T22:34:38.680+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:38 INFO Utils: Fetching spark://10.0.2.15:41161/jars/scala_2.13-0.1.0.jar to /tmp/spark-9c108a36-4e1b-4861-a47f-656f2b5d83dd/userFiles-c66e01c2-27db-4844-965f-26863ff5c703/fetchFileTemp4508001260426374827.tmp
[2024-02-06T22:34:39.185+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:39 INFO Executor: Adding file:/tmp/spark-9c108a36-4e1b-4861-a47f-656f2b5d83dd/userFiles-c66e01c2-27db-4844-965f-26863ff5c703/scala_2.13-0.1.0.jar to class loader
[2024-02-06T22:34:39.285+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:39 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 42027.
[2024-02-06T22:34:39.286+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:39 INFO NettyBlockTransferService: Server created on 10.0.2.15:42027
[2024-02-06T22:34:39.295+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:39 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-02-06T22:34:39.335+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:39 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 10.0.2.15, 42027, None)
[2024-02-06T22:34:39.422+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:39 INFO BlockManagerMasterEndpoint: Registering block manager 10.0.2.15:42027 with 434.4 MiB RAM, BlockManagerId(driver, 10.0.2.15, 42027, None)
[2024-02-06T22:34:39.524+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:39 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 10.0.2.15, 42027, None)
[2024-02-06T22:34:39.550+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:39 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 10.0.2.15, 42027, None)
[2024-02-06T22:34:41.895+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:41 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2024-02-06T22:34:41.969+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:41 INFO SharedState: Warehouse path is 'file:/home/ubuntu/airflow/spark-warehouse'.
[2024-02-06T22:34:45.512+0100] {spark_submit.py:571} INFO - Exception in thread "main" java.lang.NoClassDefFoundError: scala/collection/StringOps$
[2024-02-06T22:34:45.516+0100] {spark_submit.py:571} INFO - at FormattedData$.main(FormattedData.scala:14)
[2024-02-06T22:34:45.520+0100] {spark_submit.py:571} INFO - at FormattedData.main(FormattedData.scala)
[2024-02-06T22:34:45.571+0100] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2024-02-06T22:34:45.572+0100] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[2024-02-06T22:34:45.577+0100] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-02-06T22:34:45.577+0100] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[2024-02-06T22:34:45.578+0100] {spark_submit.py:571} INFO - at org.apache.spark.deploy.JavaMainApplication.start(SparkApplication.scala:52)
[2024-02-06T22:34:45.579+0100] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit.org$apache$spark$deploy$SparkSubmit$$runMain(SparkSubmit.scala:984)
[2024-02-06T22:34:45.579+0100] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit.doRunMain$1(SparkSubmit.scala:191)
[2024-02-06T22:34:45.580+0100] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit.submit(SparkSubmit.scala:214)
[2024-02-06T22:34:45.580+0100] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit.doSubmit(SparkSubmit.scala:90)
[2024-02-06T22:34:45.580+0100] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit$$anon$2.doSubmit(SparkSubmit.scala:1072)
[2024-02-06T22:34:45.581+0100] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit$.main(SparkSubmit.scala:1081)
[2024-02-06T22:34:45.581+0100] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit.main(SparkSubmit.scala)
[2024-02-06T22:34:45.583+0100] {spark_submit.py:571} INFO - Caused by: java.lang.ClassNotFoundException: scala.collection.StringOps$
[2024-02-06T22:34:45.584+0100] {spark_submit.py:571} INFO - at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:476)
[2024-02-06T22:34:45.584+0100] {spark_submit.py:571} INFO - at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:594)
[2024-02-06T22:34:45.585+0100] {spark_submit.py:571} INFO - at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:527)
[2024-02-06T22:34:45.585+0100] {spark_submit.py:571} INFO - ... 14 more
[2024-02-06T22:34:45.726+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:45 INFO SparkContext: Invoking stop() from shutdown hook
[2024-02-06T22:34:45.904+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:45 INFO SparkUI: Stopped Spark web UI at http://10.0.2.15:4040
[2024-02-06T22:34:45.990+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:45 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-02-06T22:34:46.200+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:46 INFO MemoryStore: MemoryStore cleared
[2024-02-06T22:34:46.208+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:46 INFO BlockManager: BlockManager stopped
[2024-02-06T22:34:46.433+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:46 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-02-06T22:34:46.450+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:46 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-02-06T22:34:46.512+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:46 INFO SparkContext: Successfully stopped SparkContext
[2024-02-06T22:34:46.514+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:46 INFO ShutdownHookManager: Shutdown hook called
[2024-02-06T22:34:46.521+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:46 INFO ShutdownHookManager: Deleting directory /tmp/spark-a70741c4-02bd-47ec-849d-71c91988f2db
[2024-02-06T22:34:46.564+0100] {spark_submit.py:571} INFO - 24/02/06 22:34:46 INFO ShutdownHookManager: Deleting directory /tmp/spark-9c108a36-4e1b-4861-a47f-656f2b5d83dd
[2024-02-06T22:34:46.961+0100] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self._application)
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 502, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master local[*] --conf spark.airflow.execution_date=2024-01-03 --name arrow-spark --class FormattedData --queue root.default --deploy-mode client /home/ubuntu/airflow/scala/target/scala-2.13/scala_2.13-0.1.0.jar. Error code is: 1.
[2024-02-06T22:34:46.966+0100] {taskinstance.py:1138} INFO - Marking task as UP_FOR_RETRY. dag_id=Competition_dag, task_id=cleanMatch_task, execution_date=20240103T000000, start_date=20240206T213416, end_date=20240206T213446
[2024-02-06T22:34:47.032+0100] {standard_task_runner.py:107} ERROR - Failed to execute job 36 for task cleanMatch_task (Cannot execute: spark-submit --master local[*] --conf spark.airflow.execution_date=2024-01-03 --name arrow-spark --class FormattedData --queue root.default --deploy-mode client /home/ubuntu/airflow/scala/target/scala-2.13/scala_2.13-0.1.0.jar. Error code is: 1.; 54944)
[2024-02-06T22:34:47.043+0100] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-06T22:34:47.071+0100] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
